### PR TITLE
Fix copy

### PR DESCRIPTION
**Pull request by TheRedPanda17**

### CODE REVIEW

File: myning/chapters/enter_mining.py [hunk 1]
```diff
@@ -80,6 +80,8 @@ def run(self):
                         print(trip_str)
                         print()
                         print(f"Mining... ({int(self.wait)} seconds left)")
+                        print(term.snow4("\nIf you don't play the game, it won't penalize you."))
```
Comment (rj-lee) [inline myning/chapters/enter_mining.py:83]:
```suggestion
                        print(term.snow4("\nIf you don't play the minigame, you won't be penalized (but you won't get bonuses either)."))
```

File: myning/objects/combat_game.py [hunk 2]
```diff
@@ -143,6 +143,9 @@ def __str__(self) -> str:
         s += "\n╚" + "═" * (8 * len(self.block_keys) + 5) + "╝\n\n"
 
         s += self._score_str
+
+        s += term.snow4("\n\nIf you don't play the game, it won't penalize you.\n")
```
Comment (rj-lee) [inline myning/objects/combat_game.py:147]: suggest same as the other

File: myning/utils/ui_consts.py [hunk 3]
```diff
@@ -22,7 +22,7 @@ class Icons(str, Enum):
     DAMAGE = "⚔️ "
     DEATH = "💀"
     DONE = "✅"
-    EXIT = "↩ "
+    EXIT = "↵ "
```
Comment (rj-lee) [inline myning/utils/ui_consts.py:25]: these look the same in github lol

File: myning/objects/combat_game.py (hunk 2 above)
Comment (TheRedPanda17) [reply myning/objects/combat_game.py:147]: You're not gonna make an easy commit for me? :D

**Review by rj-lee — APPROVED** [review]

**Pull request merged**